src/validation-messages/failures/invalid-parameter-example.ts

```diff
@@ -0,0 +1,12 @@
+import ValidationFailure from './validation-failure';
+
+class InvalidParameterExample extends ValidationFailure {
+  constructor(path: string[]) {
+    super(
+      'Parameter object can have either example or examples set, but not both.',
+      path,
+    );
```

Comment:
@RockslideJr  I would just make the validation message more helpful for now and skip the extra work for the verbose flag.  One suggestion for the message is to use the OAS spec language. 
"The 'example' field is mutually exclusive of the 'examples' field, provide one or the other or neither, but not both."